### PR TITLE
[REFACTOR]: Refactor components and useFetchNotifications

### PR DESCRIPTION
**Pull request by ZLoganZ**
This pull request includes refactoring of components and the useFetchNotifications hook. The refactoring removes unused code and updates the useFetchNotifications hook to directly return notifications.

### CODE REVIEW

**Review by quangvinht — APPROVED** [review]

File: components/workspaces/WpsMemberTable.tsx
Comment (Kreal244) [inline components/workspaces/WpsMemberTable.tsx:1]: ủa gì đây

File: components/workspaces/WpsMemberTable.tsx [hunk 1]
```diff
@@ -0,0 +1,200 @@
...
+    );
+  };
+  const handleConfirmRemoveMember = (user_id: number) => {
+    handleDeleteMember(user_id);
+  };
+
+  // Franchise
+  const handleFranchiseMemberSuccess = (data: GENERAL_RESPONSE_TYPE) => {
+    showSuccessToast(data.message);
+    if (member.owner.user_id !== currentUser.user_id && !['MA', 'HM'].includes(currentUser.role))
```
Comment (Kreal244) [inline components/workspaces/WpsMemberTable.tsx:71]: xoá cái này => if (member.owner.user_id !== currentUser.user_id && !['MA', 'HM'].includes(currentUser.role))
      router.replace(`/workspaces/${wksp_id}`);

File: components/workspaces/editMemberForm.tsx [hunk 2]
```diff
@@ -2,227 +2,27 @@ import { AxiosError, isAxiosError } from 'axios';
...
   const router = useRouter();
   const [data, setData] = useState<(ComboboxItem & { fuln: string })[]>([]);
 
+  const { user } = useMyInfo();
+  const { users } = useGetAllUsers();
+  const { members } = useGetWorkspaceMember(wksp_id);
+
   const { notificationSocket } = useStore((state) => state);
 
   useEffect(() => {
```
Comment (Kreal244) [inline components/workspaces/editMemberForm.tsx:28]: thay   useEffect(() => {
  thành   useEffect(() => {
    if (
      !_.isNil(members) &&
      members.owner.user_id !== currentUser.user_id &&
      !['MA', 'HM'].includes(currentUser.role)
    ) {
      router.replace(`/workspaces/${router.query.id}`);
    }

**Review by Kreal244 — CHANGES_REQUESTED** [review]

File: components/workspaces/WpsMemberTable.tsx (hunk 1 above)
Comment (ZLoganZ) [reply components/workspaces/WpsMemberTable.tsx:71]: sao xoa vay

**Review by Kreal244 — APPROVED** [review]

**Pull request merged**